packages/https-proxy/test/helpers/certs/ca/my-root-ca.key.pem

```diff
@@ -1,27 +1,27 @@
------BEGIN RSA PRIVATE KEY-----

```

Comment:
Change from running `git add --renormalize . ` after adding .gitattributes file to normalize line endings